checkov/terraform/module_loading/loader.py

```diff
@@ -69,7 +76,9 @@ def load(
         if os.path.exists(module_path):
             return ModuleContent(dir=module_path)
 
-        self.logger.debug(f"getting module {self.module_source} version: {self.version}")
+        self.logger.debug(f"Using {self.__class__.__name__} attempting to get module "
+                          f"{self.module_source if '@' not in self.module_source else self.module_source.split('@')[1]} "
+                          f"version: {self.version}")
```

Comment:
What happens if the `module_source` has no '@'? The access to `[1]` would fail, is there any way to avoid it?